### PR TITLE
don't fail on special characters

### PR DESCRIPTION
**Pull request by dbwpe**
To avoid failures in some environments from special characters such as é in [Montréal](http://www.telize.com/geoip/167.114.156.198), I added a strip_accents function as suggested [here](http://stackoverflow.com/questions/517923/what-is-the-best-way-to-remove-accents-in-a-python-unicode-string) using the `unicodedata` module.

It changes the character to the closest one it can use properly - In my example, é turns into e.
There might be a better way to do it, but this seems to work well.

Before, `result` failing to be set in `format_geodict()` due to é character:

```
$ python whoip.py 167.114.156.198
Traceback (most recent call last):
  File "whoip.py", line 57, in <module>
    print(format_geodict(answer))
  File "whoip.py", line 29, in format_geodict
    return result
UnboundLocalError: local variable 'result' referenced before assignment
```

After:
```
$ python whoip.py 167.114.156.198
167.114.156.198|Canada|Quebec|Montreal
```

Tested successfully on Python 2.6.5, 2.7.6, and 3.4.2.

### CODE REVIEW

File: whoip.py [hunk 1]
```diff
@@ -13,6 +14,10 @@ def getgeo(ip):
     if r.status_code == 200:
         return info
 
+def strip_accents(s):
+    return ''.join(c for c in unicodedata.normalize('NFD', s)
+      if unicodedata.category(c) != 'Mn')
```
Comment (sudomilk) [inline whoip.py]: Can we flip this around so that the condition comes before the return?

**Comment by sudomilk** [comment]
Other than the above note, this looks good! One other thing I would like is for the character variable to be renamed something more explicit like char. Long names are okay, it's just not immediately readable when c is used later. Thanks!

**Comment by sudomilk** [comment]
I'm just going to accept this and then rewrite some of it.

File: whoip.py (hunk 1 above)
Comment (dbwpe) [inline whoip.py]: Not with this approach at least - [Noted on this commit](https://github.com/sudomilk/whoip/commit/9197de97560d3ee3a2ced72d95fb11799bc4a6a5) on your master branch.

**Pull request merged**